### PR TITLE
[TE] frontend - Initial outline of new component (filter-bar)

### PR DESCRIPTION
**Pull request by ttbach**
**Filter Bar (collapsed)**
filter_bar collapsed

**Filter Bar (expanded)**
filter_bar expanded

### CODE REVIEW

File: thirdeye/thirdeye-frontend/app/styles/components/filter-bar.scss [hunk 1]
```diff
@@ -0,0 +1,4 @@
+.filter-bar {
+  width: 300px;
+  margin: 50px auto 50px 40px;
+}
```
Comment (justYves) [inline thirdeye/thirdeye-frontend/app/styles/components/filter-bar.scss]: nit: new line at EOF

File: thirdeye/thirdeye-frontend/app/pods/components/filter-bar/template.hbs [hunk 2]
```diff
@@ -0,0 +1,28 @@
+{{!-- TODO: Create separate styling for filter-bar component --}}
+<div class="entity-filter filter-bar">
+
+  {{#each config as |block|}}
+    <section class="entity-filter__group">
+      <h5 class="entity-filter__group-title">{{block.header}}
+        <i class="glyphicon"></i>
```
Comment (justYves) [inline thirdeye/thirdeye-frontend/app/pods/components/filter-bar/template.hbs]: is this a placeholder?

File: thirdeye/thirdeye-frontend/app/pods/components/filter-bar/template.hbs [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+      </h5>
+      <ul class="entity-filter__group-list {{if block.isHidden " entity-filter__group-list--hidden "}}">
```
Comment (justYves) [inline thirdeye/thirdeye-frontend/app/pods/components/filter-bar/template.hbs]: no need for space before and after the entity-filter__group-list--hidden class name `"entity-filter__group-list--hidden"`

File: thirdeye/thirdeye-frontend/app/pods/components/filter-bar/template.hbs (hunk 2 above)
Comment (ttbach) [reply thirdeye/thirdeye-frontend/app/pods/components/filter-bar/template.hbs]: Yes, it is. A lot of this will be changed in later PRs.

**Pull request merged**